### DISCUSSION TITLE
Отображение двух месячных сеток

### DISCUSSION THREAD

**Discussion by Suntechnic** [t1m0n/air-datepicker#578] [Ideas]
Не планируете ли добавить отображение двух сеток месяца, к примеру текущий + следующий?
Если нет, то что насчет опции silent во всех методах? Тогда можно бы было добиться такого отображения используя два пикера и сихронизирующую обертку над ними. В данный момент это невозможно из-за рекурсивной работки собитя onFocus

**Comment by t1m0n** [comment]
Никак руки не дойдут, а в каких методах не хватает silent? Добавить не проблема.
Если когда-нибудь буду делать, то скорее всего это и будет отдельная обертка над текущим календарем, чтобы не увеличивать размер бандла.
